Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -133,11 +135,11 @@ private void RoundEnd(RoundEndMessageEvent message)
             RestartSound = message.RestartSound;
 
             // Don't open duplicate windows (mainly for replays).
-            if (_window?.RoundId == message.RoundId)
+            if (Window?.RoundId == message.RoundId)
                 return;
 
             //This is not ideal at all, but I don't see an immediately better fit anywhere else.
-            _window = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText, message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, _entityManager);
+            Window = new RoundEndSummaryWindow(message.GamemodeTitle, message.RoundEndText, message.RoundDuration, message.RoundId, message.AllPlayersEndInfo, _entityManager);
         }
```

Comment:
Just move the window over the UI controller entirely instead of splitting it and have the UI controller handle it.

Reply:
It is unclear how I am to do this mainly due to the system/controller issue in the requested change above. 

This code has been here and like this for more than 4 years. 

This whole system works without these changes.